src/web/app/pages-instructor/instructor-course-enroll-page/instructor-course-enroll-page.component.html

```diff
@@ -108,6 +108,12 @@ <h1 id="enroll-header">Enroll Students for {{ courseId }}</h1>
                 </button>
               </div>
             </div>
+            <div class="row mt-3" style="align-items: center;">
+              <div class="box" style="background-color: #F19491;"></div><span class="mr-5">Student failed to be enrolled</span>
+              <div class="box" style="background-color: #AEEBA6;"></div><span class="mr-5">Student added</span>
+              <div class="box" style="background-color: #FCF4B3;"></div><span class="mr-5">Student modified</span>
+              <div class="box" style="background-color: #DDD;"></div><span class="mr-5">Student updated with no changes</span>
```

Comment:
Could there be a way to extract these colours here along with those in styles into the same place?